AssetTracking/AssetTracking/Controllers/OfficeItemsController.cs

```diff
@@ -6,13 +6,18 @@
 
 namespace AssetTracking.Controllers
 {
-    [Route("[controller]")]
+    //[Route("[controller]")]
     public class OfficeItemsController : Controller
     {
-        [Route("")]
+        [Route("admin/[controller]")]
         public IActionResult OfficeItems()
         {
             return View("~/Views/Admin/OfficeItems.cshtml");
         }
+        [Route("[controller]")]
```

Comment:
[nitpick] Fix formatting to add white spaces to separate the methods in your controller.